src/helperFunctions/updateTaskDetails.ts

```diff
@@ -0,0 +1,14 @@
+import fetch from '@/helperFunctions/fetch';
+import { TASKS_URL } from '@/components/constants/url';
+
+const updateTaskDetails = async (editedDetails = {}, taskID = '') => {
```

Comment:
Any reason for using `fetch` here?
We are already using `redux-toolkit` here.

Reply:
Also, you have not done error handling for this helper function.

Reply:
Will be done in future PR by migrating API call to RTK query 👍🏻

Reply:
> Any reason for using `fetch` here? We are already using `redux-toolkit` here.

Will be implementing this in future PR by migrating API call to RTK query 👍🏻

Reply:
> Also, you have not done error handling for this helper function.

Added the Error handling ✅

Reply:
@kotesh-arya Please add the ticket reference to the future work.

Reply:
Ticket for **Migration of API call to RTK Query :** https://github.com/Real-Dev-Squad/website-status/issues/503